docs/integrations/sources/xero.md

```diff
@@ -7,6 +7,18 @@ This page contains the setup guide and reference information for the Xero source
 - Tenant ID
 - Start Date
 
+**Required list of scopes to sync all streams:**
```

Comment:
Please add info how to get this scopes

Reply:
this will be requested automatically for oauth2 or this list will be needed during refresh token extraction for oss version